mongo/metadata.csv

```diff
@@ -225,3 +225,14 @@ mongodb.usage.update.time,gauge,,microsecond,,Total time spent performing update
 mongodb.usage.writeLock.countps,rate,,lock,,Number of write locks per second,0,mongodb,
 mongodb.usage.writeLock.count,gauge,,lock,,Number of write locks since server start (deprecated),0,mongodb,
 mongodb.usage.writeLock.time,gauge,,microsecond,,Total time spent performing write locks in microseconds,-1,mongodb,
+mongodb.tcmalloc.generic.current_allocated_bytes,gauge,,byte,,,0,mongodb,
+mongodb.tcmalloc.generic.heap_size,gauge,,unit,,,0,mongodb,
+mongodb.tcmalloc.tcmalloc.aggressive_memory_decommit,gauge,,unit,,,0,mongodb,
```

Comment:
- `mongodb.tcmalloc.generic.heap_size` should be in `byte`
- `aggressive_memory_decommit` seems to be boolean. Unit could be left empty since there are no suitable units for it from (https://github.com/DataDog/dogweb/blob/prod/integration/system/units_catalog.csv)